src/bionics.cpp

```diff
@@ -972,6 +972,10 @@ static bool attempt_recharge( Character &p, bionic &bio, units::energy &amount,
 void Character::process_bionic( int b )
 {
     bionic &bio = ( *my_bionics )[b];
+    if( calendar::once_every( 1_hours ) && !bio.id->fuel_opts.empty() &&
+        bio.has_flag( "AUTO_START_ON" ) && power_level <= 0_mJ ) {
```

Comment:
Sorry, but with this conditions this feature is useless. It needs to be at least every minute and somewhere between 66% - 80% of max power.

Reply:
Ok I'm setting up a menu to choose the threshold, I just need to figure out the time interval thing, because trying to turn on your CBM every minutes when you don't have fuel for it is very spammy.